charts/ocis/templates/collaboration/deployment.yaml

```diff
@@ -65,7 +65,7 @@ spec:
               value: {{ $officeSuite.insecure | quote }}
 
             - name: COLLABORATION_CS3API_DATAGATEWAY_INSECURE
-              value: {{ $.Values.insecure.ocisHttpApiInsecure | quote }}
+              value: com.github.owncloud.collaboration-{{ $officeSuite.name }}
```

Comment:
this looks off

Reply:
Yes, I didn't quite understand it at first, but now I've changed it